includes/class-wc-form-handler.php

```diff
@@ -292,20 +292,12 @@ public static function pay_action() {
 
 				do_action( 'woocommerce_before_pay_action', $order );
 
-				// Set customer location to order location
-				if ( $order->get_billing_country() ) {
-					WC()->customer->set_country( $order->get_billing_country() );
-				}
-				if ( $order->get_billing_state() ) {
-					WC()->customer->set_state( $order->get_billing_state() );
-				}
-				if ( $order->get_billing_postcode() ) {
-					WC()->customer->set_postcode( $order->get_billing_postcode() );
-				}
-				if ( $order->get_billing_city() ) {
-					WC()->customer->set_city( $order->get_billing_city() );
-				}
-
+				WC()->customer->set_props( array(
+					'billing_country'  => $order->get_billing_country() ? $order->get_billing_country()   : null,
```

Comment:
Should these be null? In other cases where something is empty, we are just setting an empty string. https://github.com/woothemes/woocommerce/pull/11821/files#diff-9ed886896d7600b13d6a12a7d0ccefc7R755 for example